modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -153,6 +155,7 @@ spec:
 
 .. Use the file to create the CatalogSource object:
 +
+[source,terminal]
```

Comment:
Any concern that these next few [source,terminal] are blue?

Reply:
Double-checked it, but I think this is just GitHub's syntax highlighting doing something funky after:

```
$ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
```

But seems like it's all good.